src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusExporter.cs

```diff
@@ -57,12 +58,16 @@ public PrometheusExporter(PrometheusExporterOptions options)
         set => this.funcExport = value;
     }
 
+    internal bool? OpenMetricsRequested { get; set; }
```

Comment:
This flag is set when the `Accept` headers are `application/openmetrics-text` and modifies the format of the timestamp values. This mirrors how the Java version works. Internally, Prometheus will request the scrape and will process the metrics based on the `Content-Type` returned by the scrape. OpenMetrics requires seconds for timetamps with a fraction portion, see: https://github.com/OpenObservability/OpenMetrics/blob/main/specification/OpenMetrics.md#timestamps-2

Reply:
The feature to send openmetrics format, and the feature to support scope info - can you separate them instead of making it in the same PR? You can do the openmetrics one first and then the scope info.

(Openmetrics format also allows us to send exemplars to prometheus)

Reply:
Happy to do that, just note that the `info` type won't be parsed by Prometheus unless it returns OpenMetrics format. Prometheus will throw an error. I can split it so that we add the `otel_scope_name` attributes in this PR, and then the `info` types in the OpenMetrics PR, so that it won't break the scraping.

Reply:
See #5107